File: java/src/test/java/com/url/ApplicationTests.java
package com.url;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

@RunWith(SpringRunner.class)
@SpringBootTest(classes = Application.class)
public class ApplicationTests {

	@Test
	public void contextLoads() {
		String[] args = new String[0];
		Application.main(args);
		System.out.println("test success!");
	}

}

File: java/src/test/java/com/example/urlshortener/ApplicationTests.kt
package com.example.urlshortener

import org.assertj.core.api.Assertions.assertThat
import com.example.urlshortener.controller.ShortUrlController
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class ApplicationTests {

	@Autowired
	private val controller: ShortUrlController? = null

	@Test
	fun contextLoads() {
		assertThat(controller).isNotNull;
	}
}
